drivers/bme680/bme680.c

```diff
@@ -39,7 +39,7 @@
 
 unsigned int bme680_devs_numof = 0;
 
-bme680_t *bme680_devs[BME680_NUMOF] = { };
+bme680_t *bme680_devs[BME680_NUMOF] = {0};
```

Comment:
```suggestion
bme680_t *bme680_devs[BME680_NUMOF] = { NULL };
```

yes there is implicit casting, but there is also a constant that makes it more obvious.